packages/nuxt/src/core/runtime/nitro/renderer.ts

```diff
@@ -199,38 +199,39 @@ export default defineRenderHandler(async (event): Promise<Partial<RenderResponse
   const nitroApp = useNitroApp()
 
   // Whether we're rendering an error page
-  const ssrError = event.node.req.url?.startsWith('/__nuxt_error')
+  const ssrError = event.path?.startsWith('/__nuxt_error')
     ? getQuery(event) as unknown as Exclude<NuxtPayload['error'], Error>
     : null
 
   if (ssrError && ssrError.statusCode) {
     ssrError.statusCode = parseInt(ssrError.statusCode as any)
   }
 
-  if (ssrError && event.node.req.socket.readyState !== 'readOnly' /* direct request */) {
+  if (ssrError && event.node && event.node.req.socket.readyState !== 'readOnly' /* direct request */) {
```

Comment:
Latest unenv has support `req.__unenv__` flag

Reply:
I'm not sure I understand 😓 Do you mean we can just do `if(ssrError)` ?